github/tests/ReplayData/Issue140.testGetFileContents.txt

```diff
@@ -2,7 +2,7 @@ https
 GET
 api.github.com
 None
-/repos/twitter/bootstrap/contents/js/bootstrap-affix.js
+/repos/twitter/bootstrap/contents//js/bootstrap-affix.js
```

Comment:
Why two `//`?

Reply:
Because the requested path is appended to `/contents/`, when a leading slash is provided, it results in two in the requested URL. Similarly, if one were to request `/js//bootstrap-affix.js`, it would result in `/repos/twitter/bootstrap/contents//js//bootstrap-affix.js`. There is no logic to remove duplicate `/` in the URL, because it does not create an issue to have them.

If you would like this addressed, I can 
1. remove the leading slash from the test, which I dont think should be done, because the change would be intended to hide expected behavior
2. update the code to remove leading slash or duplicate slashes, but that seems unnecessary as duplicate slashes do not alter behavior. Additionally, if a path with duplicate slashes were provided, it means the code alters the requested path before passing it along. The response would be unchanged, but it still feels wrong.

Reply:
Shouldn't we just remove the leading `/` in: https://github.com/PyGithub/PyGithub/blob/9d2129c7b9ff4dc79aa9e0e0ee6f67678f11e546/github/tests/Issue140.py#L51